src/SIL.LCModel/Application/ApplicationServices/XmlImportData.cs

```diff
@@ -593,7 +590,7 @@ private void CopyCustomFieldData(ICmObject cmoOld, ICmObject cmoNew)
 			Debug.Assert(mdc != null);
 			ISilDataAccessManaged sda = m_cache.DomainDataByFlid as ISilDataAccessManaged;
 			Debug.Assert(sda != null);
-			foreach (int flid in mdc.GetFields(cmoOld.ClassID, true, (int)CellarPropertyTypeFilter.All))
+			foreach (int flid in mdc!.GetFields(cmoOld.ClassID, true, (int)CellarPropertyTypeFilter.All))
```

Comment:
there's a debug assert a few lines up, let's just replace that with an exception since one would be thrown here anyway if it was null